src/app/form-entry/form-factory/form.ts

```diff
@@ -44,4 +47,62 @@ export class Form {
             });
         }
     }
+
+    get valid() {
+
+      let control = this.rootNode.control;
+
+      if (!control.valid) {
+        this.markInvalidControls(this.rootNode);
```

Comment:
remove this line to mark invalid control .Leave that option of whether to mark invalid controls to the consumer. The consumer can do something line: 
if(form.valid === false) {
  form.markInvalidControls();
}